src/DesignKit.tsx

```diff
@@ -4017,6 +4050,259 @@ const RadioGroupSection: React.FC = () => {
   );
 };
 
+const ListSection: React.FC = () => {
+  const { openToast } = useToast();
+
+  const CustomizedColumns: ListItem[] = [
+    {
+      columns: [
+        <div key={1} style={{ width: 200 }}>
```

Comment:
nit: remove keys from all these example columns

Reply:
Think this is actually necessary again now that I changed this to be an array of elements instead of objects.